### DISCUSSION TITLE
Город по умолчанию автоматически по геопозиции

### DISCUSSION THREAD

**Discussion by VychuzhaninovNA** [cdek-it/widget#50] [Q&A] [answered]
Добрый день.

Подскажите пожалуйста, можно ли, чтобы при открытии карты город по умолчанию выбирался автоматически, в зависимости от геопозиции?

Ранее на версии 2.0 виджета это было возможно, указывали просто defaultCity: 'auto'

А теперь на версии 3.0 виджета такой возможности почему то уже нет.

Параметр defaultLocation: 'auto' - не работает.

**Comment by vermorag** [comment, marked as answer]
В старших версиях виджета (3.0, 4.0) такого функционала не реализовано и не планируется.
От клиента нужно явное взаимодействие и подтверждения запроса на передачу его геопозиции.
